.cirrus.yml

```
@@ -35,7 +35,7 @@ env:
     UBUNTU_NAME: "ubuntu-2204"
 
     # Image identifiers
-    IMAGE_SUFFIX: "c5823947156488192"
+    IMAGE_SUFFIX: "c5121240166825984"
```

```suggestion
    IMAGE_SUFFIX: "c5167094042984448"
```

This is causing the build-task failures

---

Thanks! Fixed :)